tests/core/test_configuration.py

```diff
@@ -1763,3 +1763,22 @@ def test_config_paths_is_directory(self):
 
         with pytest.raises(IsADirectoryError, match="configuration file, but got a directory"):
             write_default_airflow_configuration_if_needed()
+
+    @patch.object(
+        conf,
+        "sensitive_config_values",
+        new_callable=lambda: [("mysection1", "mykey1"), ("mysection2", "mykey2")],
+    )
+    @patch("airflow.utils.log.secrets_masker.mask_secret")
+    @patch("airflow.configuration.conf.get")
+    def test_mask_conf_values(self, mock_get, mock_mask_secret, mock_sensitive_config_values):
+        mock_get.side_effect = ["supersecret1", "supersecret2"]
+        conf.mask_secrets()
+
+        mock_get.assert_any_call("mysection1", "mykey1")
+        mock_get.assert_any_call("mysection2", "mykey2")
```

Comment:
Nit, rather than masking `conf.get` please use the existing `@conf_vars` decorator.

Reply:
Sure, made the changes in latest commit